### PR TITLE
KAN-1: create todo items

### PR DESCRIPTION
**Pull request by KupoKopu**
JIRA Ticket:
https://ada-team-lukas.atlassian.net/browse/KAN-1

Implements start of application including functionality to add todo items, viewing the items and basic css styling using bootstrap.

### Coverage report
image

### Lighthouse report
image

### CODE REVIEW

File: app/__init__.py [hunk 1]
```diff
@@ -0,0 +1,11 @@
...
+from config import Config
+from flask_sqlalchemy import SQLAlchemy
+from flask_migrate import Migrate
+
+app = Flask(__name__)
+app.config.from_object(Config)
+db = SQLAlchemy(app)
+migrate = Migrate(app, db)
+
+from app import routes, models
```
Comment (luyah231) [inline app/__init__.py]: Add EOF space in all files it is missing in

File: app/routes.py [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+        todo = ToDo(task=form.task.data, description=form.description.data)
+        db.session.add(todo)
+        db.session.commit()
+        return redirect(url_for('index'))
+
+
+    return render_template('add.html', title='To Do', form=form)
+
+@app.route('/delete/<int:todo_id>', methods=['POST'])
+def delete_todo(todo_id):
```
Comment (luyah231) [inline app/routes.py]: this ticket is not for deletion. remove

File: app/models.py [hunk 3]
```diff
@@ -0,0 +1,14 @@
...
+from typing import Optional
+import sqlalchemy as sa
+import sqlalchemy.orm as so
+from app import db
+
+class ToDo(db.Model):
+    id: so.Mapped[int] = so.mapped_column(primary_key=True)
+    task: so.Mapped[str] = so.mapped_column(sa.String(32), index=True,
+                                                unique=True)
+    description: so.Mapped[str] = so.mapped_column(sa.String(256), index=True,
```
Comment (luyah231) [inline app/models.py]: remove unique on line 9, 11. todo record is fetched by id so it is not needed for task

Comment (KupoKopu) [reply app/models.py]: set to false

**Review by luyah231 — APPROVED** [review]

**Pull request merged**